ska_builder.py

```diff
@@ -28,14 +35,14 @@
 raw_build_list = open(args.build_list).read()
 BUILD_LIST = raw_build_list.split("\n")
 # Remove any that are commented out for some reason
-BUILD_LIST = [b for b in BUILD_LIST if not re.match("^\s*#", b)]
+BUILD_LIST = [b for b in BUILD_LIST if not re.match(r"^\s*#", b)]
 # And any that are just whitespace
-BUILD_LIST = [b for b in BUILD_LIST if not re.match("^\s*$", b)]
+BUILD_LIST = [b for b in BUILD_LIST if not re.match(r"^\s*$", b)]
 
-if os.uname().sysname == "Darwin":
+if platform.uname().system == "Darwin":
     os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.9"
 
-if os.uname().machine == 'i686':
+if platform.uname().machine == 'i686':
```

Comment:
Shiny branch doesn't build on 32-bit, we can cut all this right?

Reply:
Sure, though you are re-commenting on your own change here.  I don't think we actually need to review this PR.

Reply:
Though can review if we want any ska_builder improvements in master I suppose.